### ISSUE TITLE
Makes sense? ---> EQ visible in mixer strips....

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#951]
Hey there,

I was once again thinking about the mixer and given the fact that Logic & Cubase have
a very nice feature that let you have their basic EQs kinda "implemeted" (optional) 
in the mixer is pretty good and widely appreciated.  (Superb overview of all EQs in Song..)

Now I was thinking if it would be a good idea to have an option to show Radiums build in 
EQ in its mixer strips as well... Maybe also optional? 
Given the fact that Radiums EQ is also kinda vertical and slider based and
also not very wide in terms of used space, this might be worth a shot?  

Best,
tL.

**Comment by Lykdraft** [comment]
Ah yes, and somehow the mixerstrips size/widht/height once again "resets" and isn't recalled when closed and reopened... This did work a few versions ago, I am sure.

**Comment by kmatheussen** [comment]
> Ah yes, and somehow the mixerstrips size/widht/height once again "resets" and isn't recalled when closed and reopened... This did work a few versions ago, I am sure. (I am on 4.7.4 right now)

But I just fixed that, and it seems to work. Perhaps you mean the "New X-row Mixer Strips Window" options in the windows menu? Those just create new windows (you can have as many as you want visible at the same time (probably a bit useless feature though)), they don't reopen old windows.

**Comment by Lykdraft** [comment]
Yes, I mean the 1/2/3 row mixer strip size. Well, I was sure that this worked at some time. Strange.
Always resizing after closing is a bit unfortunate, but well...

> Those just create new windows (you can have as many as you want visible at the same time (probably a bit useless feature though

Multiple mixers at the same time would be an **awesome** feature and absolutely not useless
as soon as this is in: #927 (!)


_(Also an option to let the mixer "Stay On Top" would be good. )_

**Comment by Lykdraft** [comment]
oh, I didn't want to close this.

**Comment by kmatheussen** [comment]
> (Also an option to let the mixer "Stay On Top" would be good. )

You mean that the mixer, when run in a separate window, should be on top of the main window? I guess that makes sense, although I think of them as equals, where it would be just as natural to make the main window be on top of the mixer window.

**Comment by Lykdraft** [comment]
> You mean that the mixer, when run in a separate window, should be on top of the main window?

Yes. It should. 

Well, most of the other apps have **all** floating window staying on top, mixer especially...
Sometimes you have an option to **disable** mixer as floating, but most of the time you don't.
(Because mixer when open = most important = mixer stays on top. :) ) 

Digital Performer had the same behavior as radium a couple of years ago and that was the number
one feature request on motunation.com and everybody was whining because the other major players already dealt with this a long time ago.  (In that case Plugins and mixer must stay on top, was introduced in DP8 or DP9...)

(_One could also argue that it might be cool to make the mixer dockable as an alternative
instead of "stay on top". Maybe dockable instead of modular mixer area?)_

**Comment by kmatheussen** [comment]
Ok. Done.
